### PR TITLE
Styles and behavior for cloning sidebar

### PR DESCRIPTION
**Pull request by nathanstitt**
* [x] Moves menu into dedicated sidebar
* [x] Render sidebar beside the month, animate it's slide in & out
* [x] style calendar to match mockup
* [x] style sidebar assignments per mockup
* [x] create custom dragging outline behavior as shown in mockup

screen shot 2016-11-14 at 8 16 35 pm

### CODE REVIEW

File: tutor/specs/components/helpers/test-router.coffee [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class TestRouter
```
Comment (pandafulmanda) [inline tutor/specs/components/helpers/test-router.coffee]: c😎 l

File: tutor/src/components/course-calendar/past-assignments.cjsx [hunk 2]
```diff
@@ -32,23 +26,18 @@ PastAssignments = React.createClass
...
 PastAssignmentsShell = React.createClass
   render: ->
     {courseId} = @props
-
     <LoadableItem
       id={courseId}
       store={PastTaskPlansStore}
       actions={PastTaskPlansActions}
-      load={partial(PastTaskPlansActions.load, {courseId})}
-      renderLoading={ PastAssignmentsLoading }
```
Comment (pandafulmanda) [inline tutor/src/components/course-calendar/past-assignments.cjsx]: hmmm, did the `renderLoading` need to be removed? It was nice having a notice about assignments being loaded to give context for when they do suddenly appear on the page after the request comes back.

File: tutor/src/components/task-step/step-with-reading-content.cjsx [hunk 3]
```diff
@@ -28,15 +28,15 @@ ReadingStepContent = React.createClass
...
 
     {content_html, related_content} = TaskStepStore.get(id)
     {courseId} = Router.currentParams()
 
     <div className="#{stepType}-step">
-      <div
-        className="#{stepType}-content"
-        {...courseDataProps}>
+      <div className="#{stepType}-content">
+        <CourseTitleBanner courseId={courseId} />
```
Comment (pandafulmanda) [inline tutor/src/components/task-step/step-with-reading-content.cjsx]: In this particular instance, and in the reference book, the course data props was being used to style the reading content/children of this element, and not a title banner:

screen shot 2016-11-17 at 8 43 48 pm

File: tutor/src/components/course-calendar/past-assignments.cjsx (hunk 2 above)
Comment (nathanstitt) [inline tutor/src/components/course-calendar/past-assignments.cjsx]: No, that was an oversight when I converted to using simple id.

File: tutor/src/components/task-step/step-with-reading-content.cjsx (hunk 3 above)
Comment (nathanstitt) [inline tutor/src/components/task-step/step-with-reading-content.cjsx]: Opps, yeah that doesn't look good 😨 - pushed an update

**Pull request merged**